### ISSUE TITLE
garden admission webhook could make k8s unable to create pod

### ISSUE THREAD

**Issue by knightXun** [gardener/garden-setup#252]
I use garden-setup create gardener cluster on aws.After I made cert-manager webhook disable,garden-setup truely built garden successfully, but I found garden-dashboard pod cannot be created, After delete all admission webhook of garden, the dashboard can be created. I assume there are something wrong in garden webhooks.

**Comment by Diaphteiros** [comment]
Since the cert-manager as well as gardener and dashboard usually work fine with their webhooks - I just tried this on a GCP cluster and didn't have any problems - I assume that this problem is somehow connected to the configuration of your base cluster. I have also used the setup on AWS multiple times without any problems, however the last time was a few weeks ago, I'll try that again soon.
I did encounter the same problem as you had with the cert-manager webhook when trying to deploy on a VMware tanzu cluster, though. 
Not sure what the root cause of this problem is.
